pkg/hostap/monitor.go

```diff
@@ -313,6 +352,13 @@ func (m *InterfaceMonitor) writeCommand(command string) error {
 func (m *InterfaceMonitor) handleAuthenticateEvent(addr string) error {
 	m.addrMutex.Lock()
 	defer m.addrMutex.Unlock()
+	if len(m.authenticatedAddrs) == 0 {
+		m.PfInfo.Authenticated = true
+		err := m.PfInfo.ConfigureVlanStateForVFs()
```

Comment:
Don't we need an opposite of this in `handleDeauthenticateEvent`?

Reply:
yes, done.